src/ex4/server/services/item_manager.js

```diff
@@ -1,63 +1,76 @@
-import { readFile, writeToFile } from '../helper/fileHandler.js'
 import { isPokemon, isListOfPokemons, pokemonExistsInTodoList } from '../helper/pokemonHelper.js'
 import { getPokemon, getAllPokemons } from '../clients/pokemon_client.js'
+import items from '../db/models/items.cjs'
+import { Sequelize } from 'sequelize'
 
 class ItemManager {
 
     constructor() {
-        this.filePath = "./server/data/data.json" //Why is this not working??
+        this.sequelize = new Sequelize('todo_db', 'root', 'password', {
+            host: 'localhost',
+            dialect: 'mysql'
+        });
     }
 
     async getItems(req, res) {
         try {
-            const items = await readFile("./server/data/data.json")
-            res.status(200).json(items)
+            let newItems = await items(this.sequelize).findAll()
+            res.status(200).json(newItems)
         } catch (error) {
             console.log(error)
             return []
         }
     }
 
+    async changeItemStatus(req, res) {
+        try {
+            items(this.sequelize).update(
+                { status: req.body.status },
+                { where: { itemName: req.body.item } }
+            )
+            res.status(200)
```

Comment:
it's better practice to return the status instead of applying it to the res object